pandas/tests/window/test_rolling.py

```diff
@@ -1871,3 +1871,13 @@ def test_rolling_skew_kurt_floating_artifacts():
     assert (result[-2:] == 0).all()
     result = r.kurt()
     assert (result[-2:] == -3).all()
+
+
+def test_rolling_imaginary_part_of_complex():
+    # GH 46619
+
+    df = DataFrame([1j, 1 + 2j])
+    result = df.rolling(2).apply(lambda x: print(x) is None)
```

Comment:
Could you use the `arithmetic_win_operators` fixture instead to test this over all aggregation functions?